sale_elaboration/models/product_category.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    elaboration_profile_id = fields.Many2one(
+        comodel_name="product.elaboration.profile", string="Elaboration Profile"
+    )
+
+    def write(self, vals):
+        """Override update to set the elaboration profile on products."""
+        result = super().write(vals)
+        if "elaboration_profile_id" in vals:
+            products = self.env["product.product"].search(
+                [("categ_id", "in", self.ids)]
+            )
+            products.elaboration_profile_id = vals["elaboration_profile_id"]
```

Comment:
Indeed, you already have it in your recordset as `super` is already run, so this should be fine as well:
```suggestion
            products.elaboration_profile_id = self.elaboration_profile_id
```

Reply:
All atended